style.css

```diff
@@ -0,0 +1,353 @@
+}
+
+aside a:hover {
+  background-color: #36b37f;
+}
+
+aside a:active {
+  background-color: #008552;
+}
+
+div a:active {
+  background-color: #36b37f;
+}
+
+div a:hover {
+  background-color: #008552;
+}
+
+nav ul li a:hover {
+  background-color: transparent;
+  color: white;
+}
+
+section ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  margin-top: 25px;
+}
+
+.round-bttn {
+  font-size: 18px;
+  background-color: white;
+  border: 1px solid gray;
+  border-radius: 60px;
+  color: white;
+  padding: 15px 19px;
+  text-decoration: none;
+  cursor: pointer;
+  margin-top: 50px;
+}
+
+.bttn {
+  background-color: #36b37f; /* Green */
+  border: none;
+  border-radius: 4px;
+  color: white;
+  padding: 10px;
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  width: 128px;
+  margin-bottom: 15px;
+  text-align: center;
+}
+
+.p1 {
+  padding-top: 70px;
+  color: #172b4d;
+  font-size: 40px;
+  font-weight: bolder;
+  line-height: 52px;
+}
+
+.p2 {
+  color: #172b4d;
+  font-size: 20px;
+  font-weight: normal;
+  line-height: 28px;
+}
+
+.p3 {
+  padding-top: 10px;
+  color: #172b4d;
+  font-size: 40px;
+  font-weight: bolder;
+  line-height: 52px;
+}
+
+.flex-item-1 {
+  justify-content: center;
+  height: 74px;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-rows: repeat(2, 54% 46%);
+  background-color: #ebf0ee;
+  grid-column-gap: 50px;
+  min-height: 478px;
+  margin: 20px 30px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  justify-items: center;
+}
+
+.grid-container-aboutme {
+  display: grid;
+  grid-template-rows: repeat(2, 46% 54%);
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  min-height: 352px;
+  margin: 30px 30px;
+}
+
+.grid-item-2 {
+  border-radius: 8px;
+  background-color: white;
+  width: 100%;
+}
+
+.grid-item-aboutme-1 {
+  align-items: center;
+}
+
+.grid-item-aboutme-2 {
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.icon-languages {
+  min-height: 80px;
+  min-width: 80px;
+}
+
+.techs-items-basic {
+  background-color: #ebf0ee;
+  padding: 8px 12px;
+  margin: 20px 3px;
+  border-radius: 4px;
+  text-align: center;
+}
+
+.aboutme-items-basic {
+  background-color: white;
+  padding: 8px 12px;
+  border-radius: 4px;
+  text-align: center;
+  color: #36b37e;
+}
+
+.margin-aboutme-items-1 {
+  margin: 3px 5px;
+}
+
+.margin-aboutme-items-2 {
+  margin: 3px 8px;
+}
+
+.techs-items-1 {
+  flex: 4;
+}
+
+.techs-items-2 {
+  flex: 1;
+}
+
+.techs-items-3 {
+  flex: 2;
+}
+
+.techs-items-4 {
+  flex: 1;
+}
+
+.card-Title {
+  padding-top: 15px;
+}
+
+#bttn-img {
+  margin-bottom: 2px;
+}
+
+#header-welcome-link {
+  font-size: 20px;
+  font-weight: 900;
+  text-decoration: none;
+  width: 90px;
+  color: #28352f;
+}
+
+#p4 {
+  color: #36b37f;
+  font-size: 40px;
+  font-weight: bolder;
+  line-height: 52px;
+}
+
+#works-div-1 {
+  margin-bottom: 60px;
+}
+
+#aboutme-bttn {
+  margin-top: 30px;
+}
```

Comment:
- [ ] IDs should be unique on a page. This means that if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS. Kindly change your ids to classes for the above purpose.

Reply:
Done :heavy_check_mark:

Reply:
Well resolved